templates/interface/Debian.erb

```diff
@@ -9,7 +9,7 @@ allow-hotplug <%= @interface %>
 <% if @manage_address -%>
     address <%= @manage_address %>
 <% end -%>
-<% if @manage_hwaddr -%>
+<% if @manage_address and ! @manage_address.empty? -%>
```

Comment:
Why adding check on manage_address here?
It would not not work with dhcp interfaces.